edx-platform/pearson-certiport-theme/lms/templates/ccx/enrollment.html

```diff
@@ -113,6 +103,12 @@ <h2 class="hd hd-2">${_("Student List Management")}</h2>
                       <span class="fa fa-times-circle" aria-hidden="true"></span>${_("Revoke access")}
                     </button>
                   </td>
+                %else:
+                  <td>
+                    <button type="button" style="pointer-events: auto" title="${_('Can\'t revoke access on activated licensed enrollment')}" disabled>
```

Comment:
Did our Product owner suggested an alternative message or something else?. @kuipumu

Reply:
@Jacatove I haven't asked him for a suggestion regarding this message, should I ask him directly?

Reply:
@kuipumu Yes, please. You could leave a message in both slack and the ticket.

Reply:
@kuipumu Yes, please. You could leave a message in both slack and the ticket.